addons/l10n_jo/data/l10n_jo_chart_data.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+     <record id="jordan_chart_template_standard" model="account.chart.template">
+         <field name="name">Jordan Chart of Accounts - Standard</field>
+         <field name="code_digits">6</field>
+         <field name="bank_account_code_prefix">1001</field> <!-- TODO check what this is -->
```

Comment:
That should be more or less clear...

Reply:
So, normally, this prefix is used to create a first bank account from the CoA, so normally in the CoA there is no need for jo_account_100105 (same for the cash one).  I wonder however, in this CoA, what would happen if you have multiple bank or multiple cash accounts. @nadaseder @bakboka

Reply:
@jco-odoo if the user would like to add more cash or bank accounts he can use any code following 100108. This will keep the account in the "100100 - 100199" group. 

 CC @asfa-odoo